### PR TITLE
FwQueryConfig: define addresses to query in configs

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/body/values.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ class CAR:
       bus=0,
     ),
   ],
+  ecus={(0x720, None): Ecu.engine, (0x721, None): Ecu.debug},
 )
 
 FW_VERSIONS = {
```
Comment (sshane) [inline selfdrive/car/body/values.py:40]:
```suggestion
FW_VERSIONS = {
  CAR.BODY: {
    Ecu.engine: [
```

**Comment by sshane** [comment]
Hmm, maybe can't do this so easily. On Honda the same address is used for the fwdRadar and fwdCamera on a platform-by-platform basis. (0x18dab0f1)

File: selfdrive/car/body/values.py (hunk 1 above)
Comment (sshane) [reply selfdrive/car/body/values.py:40]: Hmm, maybe can't do this so easily. On Honda the same address is used for the fwdRadar and fwdCamera on a platform-by-platform basis. (0x18dab0f1)

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -853,125 +855,125 @@ class Buttons:
...
-    (Ecu.engine, 0x7e0, None): [b'\xf1\x810000000000\x00'],
+    Ecu.transmission: [b'\xf1\x87VDGMD15866192DD3x\x88x\x89wuFvvfUf\x88vWwgwwwvfVgx\x87o\xff\xbc^\xf1\x81E14\x00\x00\x00\x00\x00\x00\x00\xf1\x00bcshcm49  E14\x00\x00\x00\x00\x00\x00\x00SHI0G50NB1tc5\xb7'],
+    Ecu.fwdRadar: [b'\xf1\x00HI__ SCC F-CUP      1.00 1.01 96400-D2100         '],
+    Ecu.fwdCamera: [b'\xf1\x00HI  LKAS AT USA LHD 1.00 1.00 95895-D2020 160302'],
+    Ecu.engine: [b'\xf1\x810000000000\x00'],
   },
   CAR.KONA: {
-    (Ecu.fwdRadar, 0x7d0, None): [b'\xf1\x00OS__ SCC F-CUP      1.00 1.00 95655-J9200         ', ],
+    Ecu.fwdRadar: [b'\xf1\x00OS__ SCC F-CUP      1.00 1.00 95655-J9200         ', ],
     (Ecu.abs, 0x7d1, None): [b'\xf1\x816V5RAK00018.ELF\xf1\x00\x00\x00\x00\x00\x00\x00', ],
```
Comment (crwusiz) [inline selfdrive/car/hyundai/values.py:923]: hyundai/values

Ecu.abs

not cleanup..

**Comment by sshane** [comment]
superceeded by https://github.com/commaai/openpilot/pull/25859

**Pull request closed without merging**